Header header

Goal_msgs currentGoal
int32 m_x
int32 m_y

int8[] map

bool direction

float32 resolution
